src/lib/Core.jsx

```diff
@@ -312,7 +315,7 @@ function Core({
               userAnswer={[...userInput].pop()}
             />
           </div>
-          <div>
+          <div style={{display: 'flex',justifyContent: 'space-between',alignItems:'center'}}>
```

Comment:
add a className and add the style in `styles.css` instead of using inline style.